### ISSUE TITLE
Dark and black themes are suddenly broken as of today

### ISSUE THREAD

**Issue by Amandaville** [klaussinani/tusk#229]
Dark and black themes are suddenly broken.  They have been fine for many months but changed today.  I have toggled themes numerous times to no avail.  Note titles are correct but notes are now white background with very light grey text.  I have to revert to the standard light theme in order to read the notes.

tusk_0.23.0_amd64.deb

**Comment by Amandaville** [comment]
![tusk-broken-theme](https://user-images.githubusercontent.com/37563017/66618052-e0abf900-ec01-11e9-98ab-840fdeb903b3.png)

**Comment by jimdunkerton** [comment]
I'm seeing the same.  Screenshot show dark mode, with About dialog for system info:

![image](https://user-images.githubusercontent.com/672870/66628558-1d262780-ebf7-11e9-8e3d-05ea2baa83a5.png)

**Comment by thesidjway** [comment]
Facing the same issue, Ubuntu 16.04 and Tusk 0.23 
![image](https://user-images.githubusercontent.com/11344362/66670430-5ef6b280-ec77-11e9-826d-6e7ba3ee3f53.png)

**Comment by The-Jaeger** [comment]
I thought I was the only one with this issue

Version: 0.23.0
Electron: 4.2.2
Chrome: 69.0.3497.128
Node: 10.11.0
V8: 6.9.427.31-electron.0
OS: Linux x64 5.2.18-200.fc30.x86_64

**Comment by Amandaville** [comment]
@klaussinani 

Any update on this?

**Comment by oXvTV** [comment]
I have the same problem.

Version: 0.23.0
Electron: 4.2.2
Chrome: 69.0.3497.128
Node: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.17763

**Comment by sbelharbi** [comment]
same here
i use the old evernote.
thanks

Version: 0.23.0
Electron: 4.2.2
Chrome: 69.0.3497.128
Node: 10.11.0
V8: 6.9.427.31-electron.0
OS: Linux x64 4.17.4-041704-generic

![tusk](https://user-images.githubusercontent.com/23446793/66793980-baba7900-eecd-11e9-8a48-290c06f5965e.png)

**Comment by 2ndMessiah** [comment]
it's a common problem. same in windows.

**Comment by MarcoR83** [comment]
yes dark theme in  old evernote ui is broken, new one is also not consistent. (list/note-view) is white and hurts my eyes. Is there a chance of a 0.24 that can apply theme to new (beta) ui? :-)

**Comment by Amandaville** [comment]
@MarcoR83 what do you mean, "old evernote"?

**Comment by Amandaville** [comment]
I did a search for "old evernote" "new evernote" and came up with articles and forum posts from 5 years ago in 2014.  I don't know how that can be affecting our themes.

**Comment by 2ndMessiah** [comment]
> I did a search for "old evernote" "new evernote" and came up with articles and forum posts from 5 years ago in 2014. I don't know how that can be affecting our themes.

...It refers to the old and new UI of the evernote web client...Look it up in the setting sections..

**Comment by Amandaville** [comment]
Thanks, that was actually quite hard to find.  I tried the new version.  Black and dark themes don't work at all on the new Evernote web and only partially on the old Evernote web on tusk_0.23.0_amd64.deb.

**Comment by milan102** [comment]
I have a fix working.

1) Go to the "View" menu and click "Developer Tools" (or use the CTRL+SHIFT+I shortcut)

2) Go to the console and paste the code below. Note that this fix only works for the Black theme because I set the elements to black. You can play around with the evernoteColor variable to change it to other colors though. Set your theme to Black by pressing ALT+CTRL+E
`
// Remove borders that get in the way
document.getElementById("gwt-debug-sidebar").style.borderStyle = "none";
document.getElementById("gwt-debug-NoteView-root").style.borderStyle = "none";
document.getElementsByClassName("GJ1NOG4CO3B")[0].style.borderStyle = "none";

// Set the color to black, you can change this to any color you want
var evernoteColor = "black";

// Apply color change to the sidebar, top headers, and the note itself
document.getElementById("gwt-debug-sidebar").style.setProperty("background-color", evernoteColor, "important");
document.getElementById("gwt-debug-NotesHeaderView-root").style.setProperty("background-color", evernoteColor, "important");
document.getElementById("gwt-debug-MetaBarView-root").style.setProperty("background-color", evernoteColor, "important");
document.getElementsByClassName("GJ1NOG4CNNB")[0].style.setProperty("background-color", evernoteColor, "important");
`

**Comment by Amandaville** [comment]
I would also add that you need to press ENTER after pasting @milan102 's code into the console.  I pasted the code and closed the console without pressing ENTER afterwards and nothing happened. After a second try with pressing ENTER at the end, Tusk black theme was restored.

Huge Thanks @milan102

**Comment by sbelharbi** [comment]
@milan102 thanks,
but it seems that i have to retype the code every time i exit tusk.

for now i use sepia them... hopefully this will be fixed soon. thanks

**Comment by novdddd** [comment]
Hello everyone, I am facing the same issue, but I would like to add that we can all now support @klaussinani on GitHub Sponsors https://github.com/sponsors/klaussinani so he can work more on Tusk and the new version of Evernote. I really love this tool, and I want to seee it keep going!

GitHub Sponsors LINK: https://github.com/sponsors/klaussinani

**Comment by Amandaville** [comment]
> Hello everyone, I am facing the same issue, but I would like to add that we can all now support @klaussinani on GitHub Sponsors https://github.com/sponsors/klaussinani so he can work more on Tusk and the new version of Evernote. I really love this tool, and I want to seee it keep going!
> 
> GitHub Sponsors LINK: https://github.com/sponsors/klaussinani

I happily support developers who are maintaining useful software, but why would I support an abandoned project?  Tusk has been broken for almost 3 months without any response from the developer whatsoever.  Hopefully someone else will take over and we can support that developer for their efforts.
